src/utils/frame.js

```diff
@@ -47,6 +47,14 @@ function isExtJs(frame) {
   return /\/ext-all[\.\-]/.test(getFrameUrl(frame));
 }
 
+function isUnderscore(frame) {
+  return getFrameUrl(frame).match(/underscore/i);
+}
+
+function isLodash(frame) {
+  return getFrameUrl(frame).match(/lodash/i);
+}
+
```

Comment:
I was thinking more to avoid false positives for things like `lodashify`, `slodash`, or `underscoreize` kind of names.